ui/cypress/integration/common/CreateDeleteVolume.feature

```diff
@@ -4,3 +4,5 @@ Feature: CreateVolume
      When I go to the volume page by click the volume icon in the sidebar
      And I go to create volume page by click on Create a New Volume button
      Then I fill out the create volume form with SparseLoopDevice volume type and check if the status is Ready
+     And I delete the volume by click on the delete volume button and confirm the deletion
```

Comment:
Following on the previous comments, here's how the rest of the scenario would look like:

```gherkin
    # [...]
    And the volume "test-volume-sparse" becomes Ready
    When I select the "test-volume-sparse" volume
    And I click Delete
    And I confirm the deletion
    Then no volume is selected
    And the "test-volume-sparse" volume is removed from the list
```